public/editor-settings.toml

```diff
@@ -113,8 +113,9 @@ password = "opencast"
 #
 # Example:
 [subtitles.languages]
-# "captions/source+de" = "Deutsch"
-# "captions/source+en" = "English"
+"captions/vtt+en" = "English"
+"captions/vtt+de" = "German"
+"captions/vtt+es" = "Spanish"
```

Comment:
Shouldn't the default be the language code opencast uses in its metadata with three letters `eng, deu` instead of `en,de`

Reply:
I don't see why. They are unrelated, aren't they?

Reply:
For the sake of consistency? If we choose a language classifier we should use it everywhere, else we would have to convert them all the time. I mean we can change the used classifier for the dublincore language term

e.g. The vosk workflow decides based on the Metadata which language to use and can add that as sub flavor extension.